catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -1316,15 +1324,19 @@ export function FilesInput({
     setS3FilePickerOpen(true)
   }, [])
 
+  const isS3FilePickerEnabled = !!bucket && !!buckets?.length
```

Comment:
`bucket` is always a string (non-nullable), so what's the point checking `!!bucket`?